### PR TITLE
Add 404.ftl and 500.ftl

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/9850

### CODE REVIEW

File: en/404.ftl [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+### URL: https://www-dev.allizom.org/404/
+
+not-found-page-not-found-page-page-not-found = 404: Page Not Found
+not-found-page-sorry-we-cant-find-that-page = Sorry, we can’t find that page
+not-found-page-were-all-about-a-healthy-internet = We’re all about a healthy internet but sometimes broken URLs happen.
+not-found-page-go-back = Go Back
+
+# Variables:
+#   $about (url) - link to https://www.mozilla.org/about/
+not-found-page-learn-about-mozilla-the-non = <a href="{ $about }">Learn</a> about { -brand-name-mozilla }, the non-profit behind { -brand-name-firefox }.
```
Comment (peiying2) [inline en/404.ftl:14]: Non-profit or not-for-profit?

**Review by peiying2 — CHANGES_REQUESTED** [review]
Looks good overall. Just one question on the use of `non-profit`. 
Another question is on the outdated strings. They were marked as obsolete strings 4 years ago. Should we consider removing them while we have a chance?

**Comment by alexgibson** [comment]
@peiying2 thanks for the review. Fixed `not-for-profit` and removed the outdated strings.

**Review by peiying2 — APPROVED** [review]
Thank you for the updates. Looks good!

**Pull request merged**